### PR TITLE
Prl 2174 hearings change notification

### PR DESCRIPTION
**Pull request by swaroopa86**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java [hunk 1]
```diff
@@ -148,10 +148,8 @@ private void sendNotificationToRespondent(CaseDetails caseDetails) {
                 );
             }
         }
-
```
Comment (khurshedk) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java:151]: Remove this file if no changes are there

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/hearingmanagement/HearingsManagementController.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.prl.controllers.hearingmanagement;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
```
Comment (khurshedk) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/hearingmanagement/HearingsManagementController.java:9]: Functional integration and unit tests are missing

File: src/main/java/uk/gov/hmcts/reform/prl/services/hearingmanagement/HearingManagementService.java [hunk 3]
```diff
@@ -0,0 +1,508 @@
...
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.JURISDICTION;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.LISTED;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.POSTPONED;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.WAITING_TO_BE_LISTED;
+
+@Slf4j
+@RestController
+@SecurityRequirement(name = "Bearer Authentication")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class HearingManagementService {
```
Comment (khurshedk) [inline src/main/java/uk/gov/hmcts/reform/prl/services/hearingmanagement/HearingManagementService.java:52]: Please add service JUnit test

File: src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java (hunk 1 above)
Comment (swaroopa86) [reply src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java:151]: i will remove this change

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/hearingmanagement/HearingsManagementController.java (hunk 2 above)
Comment (swaroopa86) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/hearingmanagement/HearingsManagementController.java:9]: Functional test cannot be performed at this stage, as we need to integrate this API with fis_hmc_api. Unit tests are added in the following branch: hearing-api-demo-deploy

File: src/main/java/uk/gov/hmcts/reform/prl/services/hearingmanagement/HearingManagementService.java (hunk 3 above)
Comment (swaroopa86) [reply src/main/java/uk/gov/hmcts/reform/prl/services/hearingmanagement/HearingManagementService.java:52]: unite test case are added in following branch **hearing-api-demo-deploy**

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/hearingmanagement/HearingsManagementController.java (hunk 2 above)
Comment (swaroopa86) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/hearingmanagement/HearingsManagementController.java:9]: https://github.com/hmcts/prl-cos-api/pull/746

File: src/main/java/uk/gov/hmcts/reform/prl/services/hearingmanagement/HearingManagementService.java (hunk 3 above)
Comment (swaroopa86) [reply src/main/java/uk/gov/hmcts/reform/prl/services/hearingmanagement/HearingManagementService.java:52]: https://github.com/hmcts/prl-cos-api/pull/746

**Review by shashisk97 — APPROVED** [review]

**Pull request merged**